### DISCUSSION TITLE
Orientation Variables

### DISCUSSION THREAD

**Discussion by I-Am-Skoot** [dxx-rebirth/dxx-rebirth#722] [Ideas]
I was considering building a multi-axis cockpit interface for this, and while I am still reviewing the code, I was wondering if anyone else had run across the variables that hold the orientation of the players ship?

**Comment by vLKp** [comment]
The player's ship is an `object`, like many other things in the mine.  You want `Objects[Players[player_id].objnum].orient`.  This is a `vms_matrix` recording the ship's orientation.
